nym-vpn-core/crates/nym-vpn-api-client/src/client.rs

```diff
@@ -277,15 +306,16 @@ impl VpnApiClient {
         B: Serialize,
         E: fmt::Display + DeserializeOwned,
     {
+        let jwt = *self.vpn_api_time.read().await;
         let request = self
             .inner
             .create_patch_request(path, NO_PARAMS, json_body)
-            .bearer_auth(account.jwt(None).to_string());
+            .bearer_auth(account.jwt(jwt).to_string());
```

Comment:
When we construct the jwt. I've forgotten which way around it is, but just double check that it's the time and not the skew that is passed down here

Reply:
Yep, it's the time